### PR TITLE
change result of get_vrf_aws_details from '' to undef when value not …

### LINKED ISSUE #1060: oess_pull_aws_interfaces.pl - get_vrf_aws_details returns wrong value on failure
get_vrf_aws_details should return undef. Currently returns `''` on failure.

### PR DESCRIPTION
**Pull request by jonstout**
…found

Fixes #1060

### CODE REVIEW

File: app/oess_pull_aws_interfaces.pl [hunk 1]
```diff
@@ -177,6 +177,7 @@ sub get_vrf_aws_details {
         warn "Found\n";
         return $aws
     }
+    return;
```
Comment (ajragusa) [inline app/oess_pull_aws_interfaces.pl]: I don't understand what actually is changing here

Comment (jonstout) [reply app/oess_pull_aws_interfaces.pl]: Without `return;` the sub returns `''` which evaluates as `defined`.

Comment (jonstout) [reply app/oess_pull_aws_interfaces.pl]: Without it `update_endpoint_values` is called even when aws details can't be found.

Comment (ajragusa) [reply app/oess_pull_aws_interfaces.pl]: oh I gotcha now. Thanks

**Pull request merged**